content/using/raspberrypi/howtoguideraspberrypi.md

```diff
@@ -0,0 +1,187 @@
++++
+title = "How-To Guide for Raspberry Pi"
+description = "How to run Urbit on a Raspberry Pi."
+template = "doc.html"
+weight = 3
+[extra]
+hidetitle = "true"
++++
+
+## Introduction
+This how-to guide will walk you through setting up your own planet on a Raspberry Pi 4 in a cooling case with an optimally configured SATA SSD connected via the USB 3 port. Urbit will run great, and you’ll have enough integrated high-speed storage to run other Urbit-related applications on the same device in the future if you wish (e.g. a Bitcoin full node, Lightning node, etc).
+
+## Components
+Don’t attempt to run Urbit on anything less powerful than a Raspberry Pi 4. And never run Urbit on a microSD card or on a drive plugged into a USB 2 port. Performance will be suboptimal.
+
+The components and configuration in this how-to guide have been thoroughly tested and selected for ideal performance.
+
+**Component checklist:**
+
+- [Western Digital Blue 1TB M.2 SATA SSD](https://www.amazon.com/Blue-NAND-1TB-SSD-WDS100T2B0B/dp/B073SB2MXT/)
```

Comment:
Can we get links to this product on multiple different websites explicitly saying  (Amazon link) (Cool Other Site link) etc so that we don't have the appearance of endorsing amazon or any other seller